Kudu.Core/Deployment/Generator/OryxBuilder.cs

```diff
@@ -40,23 +42,36 @@ public override Task Build(DeploymentContext context)
 
                 FileLogHelper.Log("Running KuduSync with  " + kuduSyncCommand);
 
-                RunCommand(context, kuduSyncCommand, false, "Oryx-Build: Running kudu sync...");
+                try
+                {
+                    RunCommand(context, kuduSyncCommand, false, "Oryx-Build: Running kudu sync...");
+                } catch (CommandLineException cle)
+                {
+                    throw new DeploymentFailedException(cle);
+                }
             }
-           
+
+            string artifactPath = null;
             if (args.RunOryxBuild)
             {
                 PreOryxBuild(context);
 
                 string buildCommand = args.GenerateOryxBuildCommand(context);
-                RunCommand(context, buildCommand, false, "Running oryx build...");
+                try
+                {
+                    RunCommand(context, buildCommand, false, "Running oryx build...");
+                } catch (CommandLineException cle)
+                {
+                    throw new DeploymentFailedException(cle);
+                }
 
                 //
                 // Run express build setups if needed
                 if (args.Flags == BuildOptimizationsFlags.UseExpressBuild)
                 {
                     if (FunctionAppHelper.LooksLikeFunctionApp())
                     {
-                        SetupFunctionAppExpressArtifacts(context);
+                        artifactPath = SetupFunctionAppExpressArtifacts(context);
```

Comment:
`SetupFunctionAppExpressArtifacts` sets `packaname.txt` and `packagepath.txt` as well. We probably don't need that for consumption. Maybe we just do the packaging for Consumption, and not use `BUILD_FLAGS`.